freqtrade/rpc/rpc.py

```diff
@@ -456,7 +456,18 @@ def _rpc_count(self) -> List[Trade]:
     def _rpc_whitelist(self) -> Dict:
         """ Returns the currently active whitelist"""
         res = {'method': self._freqtrade.pairlists.name,
-               'length': len(self._freqtrade.pairlists.whitelist),
+               'length': len(self._freqtrade.active_pair_whitelist),
                'whitelist': self._freqtrade.active_pair_whitelist
                }
         return res
+
+    def _rpc_blacklist(self, add: List[str]) -> Dict:
+        """ Returns the currently active blacklist"""
+        if add:
+            self._freqtrade.pairlists.blacklist.extend(add)
```

Comment:
no validation here? I can add "lorem ipsum" as a pair ?

Reply:
yes, it's blacklist ... the only thing it does is prevent the bot from having the pair (or lorem ipsum) in the whitelist.

I can add validation - but how much? checking if it's a pair? if it's in stake currency? it's not like `/forcebuy`, where this pair is then bought immediately, which requires a different level of checking i think.

Reply:
leave it, it is not worth it

Reply:
that's not what i said ... i agree that some sort of validation would make sense - just not sure "how deep" that should go

Reply:
I imagined just checking if it is a pair. For me it is easy to type ADABTC or thinking a pair is ADA instead of ADA/BTC. documentation doesn't give an example either.